project_mis_report_budget/views/project_project_views.xml

```diff
@@ -20,7 +20,7 @@
         <field name="arch" type="xml">
             <xpath expr="//group[@name='economic_data_group']" position="after">
                 <group>
-                    <field name="last_margin" readonly="1" widget="percentage"/>
+                    <field name="last_margin" widget="percentage"/>

```

Comment:
@ChristianSantamaria now this field is editable due to new requirements... but, should be always editable?

In general, when project budget related data should be locked? Please ask LR

Reply:
After speaking with LR, we are going to leave it completely open for now. In the future, a restriction will be made so that they can only modify it if the date increases.